typing/typecore.ml

```diff
@@ -2779,12 +2784,16 @@ let name_cases default lst =
 
 (* Typing of expressions *)
 
-let unify_exp env exp expected_ty =
+(* Unify expected and actual expression type.
+   [sdesc_for_hint] is used to show integer literals in their original
+   formatting in the "Did you mean ...?" hint, see
+   [report_literal_type_constraint]. *)
+let unify_exp ?sdesc_for_hint env exp expected_ty =
```

Comment:
Same as above: there is no `actual` expression type here and it is probably better to describe the intent behind `sdesc_for_hint` rather than a specific use case.

Reply:
Done, thanks!